tests/foreman/ui/test_contentview.py

```diff
@@ -2705,7 +2705,7 @@ def test_positive_update_filter_affected_repos(session, module_org):
             cv.name, VERSION,
             'name = "{}" and version = "{}"'.format(repo1_package_name, '0.71')
         )
-        assert packages
+        assert len(packages) == 1
```

Comment:
Won't that assertion pass for empty results, too?
If I understand the problem correctly, Airgun would return something like `[{'Name', 'Version'}]` for empty table.

Reply:
@mirzal Yes it is returning `[{'Name' : ' ' ,  'Arch': ' ' }]`. 

I thought about adding validation in Airgun, I'm kind of reluctant adding such validation in an Airgun code. I don't even such validation in Airgun.

Reply:
@mirzal assert len(packages) == 1 will pass for empty results but it will even assert that there is only one result. (e.g. not more than 1 records) . hence I have even wanted to keep the next assertion to verify the correct resource.

Reply:
I see `packages` is whatever is returned from widgetastic `Table`. I think they should return empty list (at least a lot of our test code assumes it returns empty list for empty table), but this is one of base and basic classes, so I wouldn't dare to modify it myself. Changing test sounds like good solution, then.

Re `assert len(packages) == 1` - I understand it will tell us there is only one result. But it is unable to tell us if there was any result at all (next assert does that). From that point of view, I believe `assert packages` was correct and needed not to be changed.
But changing it is not wrong and definitely not blocking.